icu4j/readme.html

```diff
@@ -14,13 +14,15 @@
 <body style="background-color: rgb(255, 255, 255);" lang="EN-US"
  link="#0000ff" vlink="#800080">
 <h1>International Components for Unicode for Java (ICU4J)</h1>
-<h2>Read Me for ICU4J 62.1</h2>
-(Last Update: 2018-June-12)
+<h2>Read Me for ICU4J 62.2</h2>
```

Comment:
Should we also update the two lines with the "release-62-1" tag in this file?

Reply:
Hm, good question. I guess that depends on if the plan is to create another new tag for the update...
I'll check with Yoshito to see what the plan is.

Reply:
Okay, the plan is to indeed create a new tag. No binaries or full release, but it would be a git tag.
I'll update the readme file and force-push.
Thanks for catching this. :)

Reply:
Note: I'm only going to update the download tag, and not the API link -- since there is no plan to regenerate the API change report for this (since no APIs are changing).